### ISSUE TITLE
Database migrations not being triggered

### ISSUE THREAD

**Issue by ain** [interactive-pioneers/capistrano-docker-compose#23]
Despite that `db` role is given, database migrations are not being automatically run on deploy causing application to fail on features relating to new data.

**Comment by ain** [comment]
Command that is actually run:

```
docker-compose exec web bash -lc bundle exec rake db:migrate
```

Meaning bundle arguments getting lost without quotes, and, therefore only executing `bundle install` as by default Bundler behaviour.

**Comment by ain** [comment]
Fixed in 91709e1.
